dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Helpers/Builder.cs

```diff
@@ -76,16 +76,16 @@ public static ActionPlanTaskV2 CreateSleepScheduledActionPlanTask(string objecti
                 ThumbnailImageUrl = "https://img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/RE1s2KS?ver=0ad8",
                 TaskType = ActionPlanTaskType.Other.ToString(),
                 SignupName = "Set a consistent wake time",
-                AssociatedObjectiveIds = new Collection<string> { objectiveId.ToString() },
-                AssociatedPlanId = planId.ToString(), // Only needs to be set if adding as task after the plan
+                AssociatedObjectiveIds = new Collection<Guid?> { objectiveId },
```

Comment:
Similar question as before - should it really be nullable?

Reply:
See other reply